src/pages/forgotten-password.js

```diff
@@ -27,16 +27,20 @@ export class ForgottenPassword extends Component {
     this.setState({ flashMessage: true })
   }
 
+  renderLoaded (account) {
```

Comment:
Probably too late into it as I think I've spotted this naming convention on other PR's but I feel the naming here could be revised

Reply:
Yeah we've been following this convention for a while. Could be something to change during the proposed 3 month refactor? What would you suggest?